astroquery/nrao/tests/test_nrao.py

```diff
@@ -7,15 +7,16 @@
 import astropy.units as u
 from astropy.table import Table
 
-# Skip tests in this file until the new API is implemented:
-# https://github.com/astropy/astroquery/issues/2316
-pytest.skip(allow_module_level=True)
-
 from ... import nrao
 from astroquery.utils.mocks import MockResponse
 from ...utils import commons
 
 
+pytest.skip(allow_module_level=True,
```

Comment:
IMO these broken modules should be removed rather than keeping them around, knowingly, uselessly broken, while we wait for some incoming new implementations for the new APIs. I know the removal is beyond this PR, but if anyone interested, PRs for the removals are more than welcome.

Reply:
Cool.  I just added a skip reason here for my sanity, as I like to run

```
pytest -r sxf
```

to get a nice summary at the end to find out why tests skip, xfail or fail.

Reply:
yes, I like that summary, maybe we should do that for CI, too.

Reply:
I find it very useful for CI, especially since if you get a `requests` failure, the traceback is many pages long, and some modules log up a storm too.  A summary at the bottom of the pytest output is really nice, as above in https://github.com/astropy/astroquery/pull/2526#issuecomment-1245238149

Reply:
Yes. I always use that one-liner per failure output for the failures, and didn't know that xfails and skips can be harvested, too, it's indeed really useful for spotting things that could be cleaned up (I'm certain we're skipping a few things that should not be skipped any more).